blog/2015/09/teaching-to-the-workflow.html

```diff
@@ -0,0 +1,91 @@
+---
+layout: blog
+root: ../../..
+author: Naupaka Zimmerman
+title: "Teaching to the Workflow"
+date: 2015-09-15
+time: "10:00:00"
+category: ["Teaching"]
+---
+<!-- start excerpt -->
+<p>
+  "Teaching to the test" has a deservedly bad reputation,
+  but what about "teaching to the workflow"?
+  In <a href="http://www.esajournals.org/doi/abs/10.1890/ES14-00402.1">a recent paper</a>,
+  my colleagues and I sketched three examples of possible open science workflows.
```

Comment:
Change to, "A group of us came together at the [NCEAS Open Science Codefest](http://nceas.github.io/open-science-codefest/) last year and put together [a paper on Open Science in Ecology](http://www.esajournals.org/doi/abs/10.1890/ES14-00402.1). In it, we sketch three examples..."